zebra-state/src/service/check/anchors.rs

```diff
@@ -103,6 +108,16 @@ pub(crate) fn anchors_refer_to_earlier_treestates(
         // Sapling Spends
         //
         // MUST refer to some earlier block’s final Sapling treestate.
+        //
+        // # Consensus
+        //
+        // > The anchor of each Spend description MUST refer to some earlier
+        // > block’s final Sapling treestate. The anchor is encoded separately
+        // > in each Spend description for v4 transactions, or encoded once and
+        // > shared between all Spend descriptions in a v5
+        // > transaction.
```

Comment:
This consensus rule is naturally implemented in different parts of the codebase. The implementation below effectively deals only with the first sentence of the consensus rule. What's the best way to deal with a situation like this?

I can think of either splitting the consensus rule and mentioning only the relevant part or quoting the whole consensus rule in multiple places in the codebase. None of that seems ideal.